pkg/pdapi/pdapi.go

```diff
@@ -93,7 +93,7 @@ type PDClient interface {
 }
 
 var (
-	healthPrefix           = "pd/health"
+	healthPrefix           = "pd/api/v1/health"
```

Comment:
which version of PD was this API introduced?

Reply:
v3.1.0

Reply:
We can merge this to master and make it clear that TiDB <=v3.0 is not supported from v1.4.0. @csuzhangxc